api/logs/v1/http.go

```diff
@@ -150,6 +151,7 @@ func NewHandler(read, tail, write, rules *url.URL, rulesReadOnly bool, upstreamC
 	}
 
 	r := chi.NewRouter()
+	r.Use(server.StripTenantPrefix("/api/v1/logs"))
```

Comment:
Maybe we can do logs and traces separately as a follow-up PR?

Reply:
I prefer a single one, as this started, to make it easier to deploy/revert.